tianshou/data/utils.py

```diff
@@ -37,6 +37,8 @@ def to_torch(x: Union[torch.Tensor, dict, Batch, np.ndarray],
             x[k] = to_torch(v, dtype, device)
     elif isinstance(x, Batch):
         x.to_torch(dtype, device)
+    elif isinstance(x, np.generic):
```

Comment:
Maybe it would be even better to use `else` instead of `isinstance(x, np.generic)`, to act as fallback. What do you think @Trinkle23897 ?